.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: Automatic Release
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Export tag to env
+        run: |
+          GIT_TAG=$(echo ${GITHUB_REF#refs/*/} | cut -d 'v' -f 2)
+          echo "GIT_TAG=$GIT_TAG" >> $GITHUB_ENV
+      
+      - name: Create release file
+        run: |
+          sbt "set Universal/packageName := \"daffodil-debugger-${{env.GIT_TAG}}\"" universal:packageBin
```

Comment:
There is already a plugin in sbt that sets the version using git describe.

Did you add `set Universal/packageName := \"daffodil-debugger-${{env.GIT_TAG}}\"` for some other reason?

Reply:
well would that set the name as `daffodil-debugger-v0.0.8` then? I was mostly using it this way so that the `v` was removed

Reply:
I thought the v prefix was dropped by default...

https://github.com/sbt/sbt-git